src/libs/Notification/PushNotification/subscribePushNotification/index.ts

```diff
@@ -36,6 +39,20 @@ function getLastUpdateIDAppliedToClient(): Promise<number> {
     });
 }
 
+function buildOnyxUpdatesFromServer({onyxData, lastUpdateID, previousUpdateID}: {onyxData: OnyxServerUpdate[]; lastUpdateID: number; previousUpdateID: number}) {
```

Comment:
I guess we only ended up using this function once? If so can we just get rid of it and move the code back into the `applyOnyxUpdates` function?